### ISSUE TITLE
Awesome

### ISSUE THREAD

**Issue by joemcbride** [matoom/isharon-view#1]
Just wanted to say - thanks for this!  I had thought of doing something like this myself though never got around to it.

**Comment by matoom** [comment]
Yep, Isharon has such great resources but i was having trouble with the XLS format as well. Still need to polish out some issues but i have to try and convince Isharon to put something like this up on his site to be able to skip the downloading part and open his spreadsheet directly in browser.
